Sources/Core/GraphRequest/GraphRequestProtocol.swift

```diff
@@ -59,7 +59,7 @@ extension GraphRequestProtocol {
 
    - parameter completion: Optional completion closure that is going to be called when the connection finishes or fails.
    */
-  public func start(completion: ((httpResponse: NSHTTPURLResponse?, result: GraphRequestResult<Self>) -> Void)? = nil) {
+  public func start(_ completion: @escaping GraphRequestConnection.Completion<Self>) {
```

Comment:
Shouldn't this be `GraphRequestConnection<Self>.Completion`? The typealias itself doesn't have the generic paramter, `GraphRequestConnection` does, no?